packages/libhdf5/meta.yaml

```
@@ -0,0 +1,32 @@
+package:
+  name: libhdf5
+  version: 1.12.1
+
+source:
+  sha256: e6dde173c2d243551922d23a0387a79961205b018502e6a742acb30b61bc2d5f
+  url: https://github.com/HDFGroup/hdf5/archive/refs/tags/hdf5-1_12_1.tar.gz
+
+requirements:
+  run:
+    - zlib
+
+build:
+  library: true
+  script: |
+    mkdir -p build;
+    cd build \
+        && LDFLAGS="-s NODERAWFS=1" emcmake cmake ../ \
+        -DCMAKE_CROSSCOMPILING_EMULATOR="$PYODIDE_ROOT/emsdk/emsdk/node/14.18.2_64bit/bin/node" \
+        -DCMAKE_INSTALL_PREFIX=hdf5 \
+        -DH5_HAVE_GETPWUID=0 \
+        -DH5_HAVE_SIGNAL=0 \
+        -DBUILD_SHARED_LIBS=0 \
+        -DBUILD_STATIC_LIBS=1 \
```

Let's give this a try?
```suggestion
        -DBUILD_SHARED_LIBS=1 \
        -DBUILD_STATIC_LIBS=0 \
```